scanapp/tasks.py

```diff
@@ -188,3 +229,17 @@ def fill_unfilled_scan_model(scan, files_count, scancode_notice, scancode_versio
     scan.scancode_version = scancode_version
     scan.save()
     return scan
+
+
+def parse_url(URL):
+    """
+        Parses the URL and checks if it's a git URL. If it is a git URL then the flag is set to 1.
+    """
+    flag = 0
+    g = GitURL(URL)
+
+    try:
```

Comment:
use a boolean instead. None evals to False alright.  You could replace this whole function with
`return GitURL(url).is_a('github')`
And if this is a onliner, why a function? also the doc string is misleading if you only handle github urls and not generally git-related urls